src/me/hkgumbs/tictactoe/main/java/formatter/ThreeCharacterSlot.java

```diff
@@ -0,0 +1,21 @@
+package me.hkgumbs.tictactoe.main.java.formatter;
+
+import me.hkgumbs.tictactoe.main.java.board.Board;
+
+public class ThreeCharacterSlot implements SlotRepresentation {
+
+    @Override
+    public String compile(Board.Mark mark, int position) {
+        if (mark != null)
+            return " " + mark.toString() + " ";
+        else if (position < 10)
+            return "(" + Integer.toHexString(position) + ")";
+        else
+            return "...";
```

Comment:
Heh, we were commenting at the same time. Seems like the board size may need to determine the # of characters in the printed representation.